demos/README.md

```diff
@@ -265,7 +265,7 @@ For example, for the **Debug** configuration, go to the project's
 **Configuration Properties** to the **Debugging** category and set the `PATH`
 variable in the **Environment** field to the following:
 
-```sh
+```
```

Comment:
```suggestion
```bat
```

Reply:
Why? It's not a command.

Reply:
It is. `PATH=<INSTALL_DIR>\deployment_tools\inference_engine\bin\intel64\Debug;<INSTALL_DIR>\opencv\bin;%PATH%` exports a new value for `PATH` var. Except for `<INSTALL_DIR>` part. But there is a similar update for `<INSTALL_DIR>\bin\setupvars.bat` above.

Reply:
Read the text above. This is supposed to be a value for the user to set in a configuration dialog, not a command.

It's not even a valid Batch command. :-) You can't do variable assignments with just an equals sign.

Reply:
My bad